mpi4jax/_src/xla_bridge/__init__.py

```diff
@@ -13,46 +9,10 @@
 else:
     HAS_GPU_EXT = True
 
-
-def is_truthy(str_val):
-    return str_val.lower() in ("true", "1", "on")
-
-
-def is_falsy(str_val):
-    return str_val.lower() in ("false", "0", "off")
-
-
-enable_logging = is_truthy(os.environ.get("MPI4JAX_DEBUG", ""))
-mpi_xla_bridge.set_logging(enable_logging)
-
-
-if HAS_GPU_EXT:
-    gpu_copy_behavior = os.environ.get("MPI4JAX_USE_CUDA_MPI", "")
-
-    if is_truthy(gpu_copy_behavior):
-        has_cuda_mpi = True
-    elif is_falsy(gpu_copy_behavior):
-        has_cuda_mpi = False
-    else:
-        has_cuda_mpi = False
-        warn_msg = (
-            "Not using CUDA-enabled MPI. "
-            "If you are sure that your MPI library is built with CUDA support, "
-            "set MPI4JAX_USE_CUDA_MPI=1. To silence this warning, "
-            "set MPI4JAX_USE_CUDA_MPI=0."
```

Comment:
You moved the warning somewhere else or you removed it?

Reply:
It's here now:

https://github.com/PhilipVinc/mpi4jax/pull/62/files#diff-20bdfe84be6f1fd1effb18bb019f53d6a47fee9adb3a039a062899f775bcc191R65-R89